modules/system/assets/ui/docs/select.md

```diff
@@ -68,11 +68,28 @@ Use the `data-handler` attribute to source the select options from an AJAX handl
     data-minimum-input-length="2"
     data-ajax--delay="300"
     data-request-data="foo: 'bar'"
-    ></select>
+></select>
 ```
 
 The AJAX handler should return results as an array.
 
+```php
+public function onGetOptions()
+{
+    $results = [
+        [
```

Comment:
@DanHarrin  just change brackets here from `[` to `{` and `],` to `},`.

Reply:
@w20k but its PHP?

Reply:
@w20k the keyed array gets parsed to `{}` when it's passed to JS

Reply:
@DanHarrin Sorry, still not waked up 😄 My bad, will provide you with the tips for JS in a minute)